Source/HelixToolkit.Wpf.SharpDX/Controls/Effects.cs

```diff
@@ -151,42 +142,40 @@ private void InitEffects()
  new[] 
                 { 
                     // put here the techniques which you want to use with this effect
-                    Techniques.RenderPhong, 
-                    Techniques.RenderBlinn, 
-                    Techniques.RenderCubeMap, 
-                    Techniques.RenderColors, 
-                    Techniques.RenderDiffuse,
-                    Techniques.RenderPositions,
-                    Techniques.RenderNormals,
-                    Techniques.RenderPerturbedNormals,
-                    Techniques.RenderTangents, 
-                    Techniques.RenderTexCoords, 
-                    Techniques.RenderWires, 
-                    Techniques.RenderLines,
-                    Techniques.RenderPoints,
-                    Techniques.RenderBillboard,
+                    renderTechniquesManager.RenderTechniques[DefaultRenderTechniqueNames.Phong],

```

Comment:
`RenderTechinque` lookup is now done on the `RenderTechniques` dictionary, NOT on the static `Techniques` class.